src/klocek.cpp

```diff
@@ -60,6 +60,23 @@ std::vector<std::vector<bool>> Klocek::dajTablice()
 }
 std::vector<std::vector<bool>> Klocek::obroc(KierunekObrotu kierunekObrotu = PRAWO)
 {
-    // TODO
-    return {{}};
+    if(kierunekObrotu == LEWO){
+        for(int i = 0 ; i < 2; i++){
+            obroc();
+        }
+    }
```

Comment:
Należy to usunąć i normalnie dodać ifa w środku obracania, żeby nie było takiego spaghetti.